Makefile

```diff
@@ -0,0 +1,20 @@
+serve:
+	@ cd docs/html && if python -V 2>&1 | grep 'Python 2' > /dev/null; then \
+		python -m SimpleHTTPServer ;\
+	else \
+		python -m http.server ;\
+	fi
+
+clean:
+	find . -name build_out|xargs rm -rf
+	find . -name __pycache__|xargs rm -rf
+	find . -name "*.pyc"|xargs rm -rf
+	find . -name "*.swp"|xargs rm -rf
```

Comment:
This breaks vim's recovery and locking mechanisms in its default configuration.

Reply:
I make no claims these commands are appropriate, they're lifted straight from the pre-existing `clean` script. The point was to have as small a change as possible rather than try to change anything.